website/content/en/v1.0/tasks/managing-amis.md

```diff
@@ -17,138 +28,110 @@ See [How do I upgrade an EKS Cluster with Karpenter]({{< relref "../faq/#how-do-
+
+Karpenter offers you various controls to ensure you don't take on too much risk as you rollout new versions of AMIs to your production clusters. Below shows how you can use these controls:
+
+* [Pinning AMIs]({{< relref "#pinning-amis" >}}): If workloads require a particluar AMI, this control ensures that it is the only AMI used by Karpenter. This can be used in combination with [Testing AMIs]({{< relref "#testing-amis" >}}) where you lock down the AMI in production, but allow the newest AMIs in a test cluster while you test your workloads before upgrading production.
+* [Testing AMIs]({{< relref "#testing-amis" >}}): The safest way for ensuring that a new AMI doesn't break your workloads is to test it before putting it into production. This takes the most effort on your part, but most effectively models how your workloads will run in production, allowing you to catch issues ahead of time. Note that you can sometimes get different results from your test environment when you roll a new AMI into production, since issues like scale and other factors can elevate problems you might not see in test. Combining this with other controls like [Using Disruption Budgets]({{< relref "#using-disruption-budgets" >}}) can allow you to catch problems before they impact your whole cluster.
+* [Using Disruption Budgets]({{< relref "#using-disruption-budgets" >}}): This option can be used as a way of mitigating the scope of impact if a new AMI causes problems with your workloads. With Disruption budgets you can slow the pace of upgrades to nodes with new AMIs or make sure that upgrades only happen during selected dates and times (using `schedule`). This doesn't prevent a bad AMI from being deployed, but it allows you to control when nodes are upgraded, and gives you more time to respond to rollout issues.
+
+## Controls
+
+The following lays out the controls you have to impact Karpenter’s behavior as it relates to how nodes are created and AMIs are consumed.
+
+### Pinning AMIs
+
+When you configure the [**EC2NodeClass**]({{< relref "../concepts/nodeclasses" >}}), you are required to configure which AMIs you want Karpenter to select on using the `amiSelectorTerms` field. When pinning to a specific `id`, `name` or an `alias` that contains a fixed version, Karpenter will only select on a single AMI and won't automatically upgrade your nodes to a new version of an AMI. This prevents a new and potentially untested AMI from replacing existing nodes when those nodes are terminated.
+).
+These examples show three different ways to identify the same AMI:
 
 ```yaml
+# Using alias
 amiSelectorTerms:
-- tags:
-    karpenter.sh/discovery: "${CLUSTER_NAME}"
-    environment: prod
-- name: al2023-ami-2023.3.20240219.0-kernel-6.1-x86_64
+- alias: al2023@v20240219
```

Comment:
It's probably worth mentioning somewhere that this isn't a true pin, and will still result in drift during kubernetes version upgrades.

Reply:
Ah, I forgot about that bit -- still makes sense to me that we would still be comfortable recommending it, though -- since the only time that an upgrade is going to occur to the AMI is going to be when there is a user-initiated action to bump the K8s version up